vets_json_schema.gemspec

```diff
@@ -12,4 +12,4 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'multi_json', '~> 1.0'
   gem.add_dependency 'script_utils', '0.0.4'
-end
```

Comment:
@nihil2501 Can you please add the empty line back to this file?

Reply:
@RachalCassity I undid that edit.